internal/profiling/goid_modified.go

```diff
@@ -52,17 +52,30 @@ import (
 //    special Go code that's both architecture-specific and go version-specific
 //    (a quadratic number of variants to maintain).
 //
-// 4. This approach, which requires a simple two-line modification to the Go
+// 4. This approach, which requires a simple two-line modification [1] to the Go
```

Comment:
Hey, that's four lines!  ;)

Seriously though, it's super minor but please delete "two-line" -- "simple modification" and a reference is sufficient.

Reply:
Done.